source/xlsxreader.d

```diff
@@ -90,20 +207,20 @@ struct Sheet {
 		return app.data;
 	}
 
-	void printTable() {
+	void printTable() const {
 		writeln(this.toString());
 	}
 
 	// Column
 
-	Iterator!T getColumn(T)(size_t col, size_t start, size_t end) {
-		auto c = this.iterateColumn!T(col, start, end);
-		return Iterator!T(c.array);
+	Iterator!T getColumn(T)(size_t col, size_t startRow = 0, size_t endRow = size_t.max) {
```

Comment:
The default value here will in the worst case result in a gigantic allocation. Please remove them.

Reply:
There is the maxPos member, but that is also bad. The user has to choose.

Reply:
So what about adding an getColumn-overload or another member function that infers endRow from maxPos?

Reply:
IMO that only works until it does not, and you never know when it doesn't. A bit of pain now is better than a lot later.

Reply:
1. Are you saying size_t startRow = 0 is a bad idea to? Why?
2. I don't see how adding a function, say `getFullColumn` or `getWholeColumn`, defaulting `start` to `0` and `end` to `maxPos+1`, can go wrong. Can you elaborate?

Reply:
> The default value here will in the worst case result in a gigantic allocation. Please remove them.

Can you elaborate on how that would happen considering the presence of 

`this.endRow = endRow != size_t.max ? endRow : sheet.maxPos.row + 1;`

ColumnUntyped constructor.

Reply:
For the record, the code in the upcoming extract plugin will not know the column count in advance and will have to make use of `maxPos` nevertheless.

Reply:
I've removed these default inits at https://github.com/symmetryinvestments/xlsxreader/pull/16/commits/4041ad121fdfb0ab59cb16b87d76f9233797bccf. Ready to merge now?